hubspot3/tickets.py

```diff
@@ -316,7 +316,10 @@ def _merge_changes_with_history(self,
                     min_time_index = min(range(len(time_diffs)),
                                          key=time_diffs.__getitem__)
                     if time_diffs[min_time_index] <= max_time_diff_ms:
-                        value = versions[min_time_index]["value"]
+                        if "value" in versions[min_time_index]:
+                            value = versions[min_time_index]["value"]
+                        else:
+                            value = None
```

Comment:
You can use `dict.get`: `value = versions[min_time_index].get("value")`

Reply:
keep forgetting that there is a good way to do that...